### ISSUE TITLE
There is no way to create DNS record with 2 or more IP 

### ISSUE THREAD

**Issue by jia-jerry** [terraform-providers/terraform-provider-alicloud#340]
### Terraform Version
Terraform v0.11.6
+ provider.alicloud v1.10.0

### Affected Resource(s)
- alicloud_dns_record
 
Current config to create a dns record example is:
```
resource "alicloud_dns_record" "record" {
  name = "example.com"
  host_record = "www"
  type = "A"
  value = "1.2.3.6"
}
```
How to bind 2 IPs to the same domain?

**Comment by bittopaz** [comment]
I guess it is the only way as below

```
resource "alicloud_dns_record" "record1" {
  name = "example.com"
  host_record = "www"
  type = "A"
  value = "1.2.3.6"
}

resource "alicloud_dns_record" "record2" {
  name = "example.com"
  host_record = "www"
  type = "A"
  value = "1.2.3.7"
}
```

**Comment by xiaozhu36** [comment]
HI @jia-jerry The new release 1.25.0 has fixed this issue. Please check.

**Comment by xiaozhu36** [comment]
Hi @jia-jerry I think this issue has been fixed by release 1.26.0. Please update it and try again.

**Comment by xiaozhu36** [comment]
Closed by no reply.
